pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.9.35"
+version = "0.9.35.dev-38"
```

Comment:
@sid should we update to "0.9.36" in this pr?

Reply:
no, leave it as is. Non-dev releases should be done through the release process https://www.notion.so/ml-infra/Release-Truss-Versions-bc78f4e5b62b4f4389bc46748db955db?pvs=4